frontend/src/index/App.tsx

```diff
@@ -55,14 +56,15 @@ function App() {
                 <BrowserRouter>
                     <div className={"floating-navbar"}>
                         <nav>
-                        <NavLink to={"/"}>Pferde</NavLink>
+                        <NavLink to={"/pferde/ueberblick"}>Pferde</NavLink>
                             <NavLink to={"/"}>Einstaller</NavLink>
                             <NavLink to={"/lager/ueberblick"}>Lager</NavLink>
                             <NavLink to={"/"}>Account</NavLink>
                         </nav>
                     </div>
                     <Routes>
                         <Route path={"/lager/ueberblick"} element={<StockOverview/>}></Route>
+                        <Route path={"/pferde/ueberblick"} element={<HorseOverview/>}></Route>
```

Comment:
Sure that the URLS shouldn't be in english?

Reply:
Since they are displayed in the browser i'll prefer them in German